JustBrewIt/Pages/Search.cshtml.cs

```diff
@@ -16,44 +16,41 @@ public class SearchModel : PageModel
         public string BreweryType { get; set; }
         [BindProperty]
         public string BreweryCity { get; set; }
-        public bool isSearchCity { get; set; }
+        public bool isSearchCity = false;
 
-        public bool isSearchType { get; set; }
+        public bool isSearchType = false;
         public string Url { get; set; }
-
-        public void OnGet()
-        {
-            isSearchCity = false;
-            isSearchType = false;
-        }
+        public string Error;
         
         public void OnPost()
         {
             string city = BreweryCity;
             string type = BreweryType;
-
-           if(city == null && type != null) { 
-
-            Url = "https://api.openbrewerydb.org/breweries?by_state=ohio&by_type=" + BreweryType;
-           }
-           else if(city != null && type != null)
-            {
-                Url = "https://api.openbrewerydb.org/breweries?by_type=" + type + "&by_city=" + city;
+            Url = "https://api.openbrewerydb.org/breweries?by_city=" + city;
+            if (String.IsNullOrEmpty(city) && !String.IsNullOrEmpty(type)) {
+                Url = "https://api.openbrewerydb.org/breweries?by_state=ohio&by_type=" + BreweryType;
             }
-            else
+            else if(!String.IsNullOrEmpty(city) && !String.IsNullOrEmpty(city))
```

Comment:
Modified check with != null with String function IsNullOrEmpty to handle empty strings